options/glibc/include/netipx/ipx.h

```diff
@@ -0,0 +1,19 @@
+#ifndef _NETIPX_IPX_H
+#define _NETIPX_IPX_H
+
+#include <mlibc-config.h>
+
+typedef struct ipx_config_data {
+	unsigned char ipxcfg_auto_select_primary;
+	unsigned char ipxcfg_auto_create_interfaces;
+} ipx_config_data;
+
+#if defined __linux__ || defined __MLIBC_LINUX_OPTION
+#include <linux/sockios.h>
+
+#define SIOCAIPXITFCRT (SIOCPROTOPRIVATE)
+#define SIOCAIPXPRISLT (SIOCPROTOPRIVATE + 1)
+#define SIOCIPXCFGDATA (SIOCPROTOPRIVATE + 2)
+#endif
+
+#endif // _NETIPX_IPX_H
```

Comment:
Don't include C++-style comments in public headers

Reply:
Makes sense, but we should probably check the entire codebase for that then.

Reply:
Will do at a later date (as mentioned). The only reason I'm pedantic about this is because it came up as a problem when dealing with C89 programs